modules/ROOT/pages/scopes-and-collections-config.adoc

```diff
@@ -28,14 +28,22 @@ Keep in mind that the provided examples are optional.
 You have these options to configure Scopes and Collections Sync for Gateway:
 
 * *custom scope/custom collection (the option we cover here)* +
-Gain maximum flexibility in organizing and accessing your data by customizing it to suit your needs.
+You can define 1 custom scope per database with up to 1000 custom collections in release 3.1. +
+
+Scopes and Collections are optional. 
+Default scopes and collections will be applied automatically if non are specified.
 
 * *default scope/custom collection* +
-Create a custom collection with the default scope can lead to efficient data organization and quicker access control management.
+You can choose the default scope with up to 1000 custom collections in release 3.1.
+
+Scopes and Collections are optional. 
+Default scopes and collections will be applied automatically if non are specified.
 
 * *default scope/default collection* +
-Work within the default Scopes and Collections. 
-This makes managing data easy and simple without creating extra scopes or collections. 
+You can choose the default scope with default collections in release 3.1.
```

Comment:
This wording is different, and again "default collections" is potentially misleading